content/en/templates/shortcode-templates.md

```diff
@@ -155,7 +155,7 @@ While you can create shortcode templates that accept both positional and named p
 
 You can also use the variable `.Page` to access all the normal [page variables][pagevars].
 
-A shortcodes can also be nested. In a nested shortcode, you can access the parent shortcode context with [`.Parent` variable][shortcodesvars]. This can be very useful for inheritance of common shortcode parameters from the root.
+Shortcodes can also be nested. In a nested shortcode, you can access the parent shortcode context with the [`.Parent` shortcode method][parent]. This can be very useful for inheritance of common shortcode parameters from the root.
```

Comment:
```suggestion
Shortcodes can also be nested. In a nested shortcode, you can access the parent shortcode context with the [`.Parent`] shortcode method. This can be very useful for inheritance of common shortcode parameters from the root.
```